R tests/test_CodeId.py::test_CodeId[5001-5000-UPC-A0]
 test_CodeId.py:17: in test_CodeId
     assert output == device.scanner.scan(symbology_num, subtest, barcode[0], size, use)
 ../lib/scanner.py:19: in scan
     msg = 'lipc-set-prop com.lab126.powerd flIntensity {}\r'.format(level)
 E   NameError: global name 'level' is not defined
F tests/test_CodeId.py::test_CodeId[5001-5000-UPC-A0]
 test_CodeId.py:17: in test_CodeId
     assert output == device.scanner.scan(symbology_num, subtest, barcode[0], size, use)
 ../lib/scanner.py:19: in scan
     msg = 'lipc-set-prop com.lab126.powerd flIntensity {}\r'.format(level)
 E   NameError: global name 'level' is not defined
R tests/test_CodeId.py::test_CodeId[5001-5000-UPC-A1]
 test_CodeId.py:17: in test_CodeId
     assert output == device.scanner.scan(symbology_num, subtest, barcode[0], size, use)
 ../lib/scanner.py:19: in scan
     msg = 'lipc-set-prop com.lab126.powerd flIntensity {}\r'.format(level)
 E   NameError: global name 'level' is not defined
E tests/test_CodeId.py::test_CodeId[5001-5000-UPC-A1]
 ../conftest.py:183: in logfolder
     os.makedirs(logfolder)
 ../../venv27/lib/python2.7/os.py:157: in makedirs
     mkdir(name, mode)
 E   OSError: [Errno 17] File exists: 'logs/21062022_104424.478486/21342522506311'
